src/coreclr/scripts/superpmi_asmdiffs_summarize.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+#
+## Licensed to the .NET Foundation under one or more agreements.
+## The .NET Foundation licenses this file to you under the MIT license.
+#
+##
+# Title: superpmi_asmdiffs_summarize.py
+#
+# Notes:
+#
+# Script to summarize issues found from all partitions and print them on console.
+#
+################################################################################
+################################################################################
+
+import argparse
+import os
+from coreclr_arguments import *
+
+parser = argparse.ArgumentParser(description="description")
+
+parser.add_argument("-diff_summary_dir", help="Path to diff summary directory")
+parser.add_argument("-arch", help="Architecture")
+parser.add_argument("-platform", help="OS platform")
+
+def setup_args(args):
+    """ Setup the args.
+
+    Args:
+        args (ArgParse): args parsed by arg parser
+
+    Returns:
+        args (CoreclrArguments)
+
+    """
+    coreclr_args = CoreclrArguments(args, require_built_core_root=False, require_built_product_dir=False,
+                                    require_built_test_dir=False, default_build_type="Checked")
+
+    coreclr_args.verify(args,
+                        "diff_summary_dir",
+                        lambda diff_summary_dir: os.path.isdir(diff_summary_dir),
+                        "diff_summary_dir doesn't exist")
+
+    coreclr_args.verify(args,
+                        "arch",
+                        lambda unused: True,
+                        "Unable to set arch")
+
+    coreclr_args.verify(args,
+                        "platform",
+                        lambda unused: True,
+                        "Unable to set platform")
+
+    return coreclr_args
+
+
+def main(main_args):
+    """Main entrypoint
+
+    Args:
+        main_args ([type]): Arguments to the script
+    """
+
+    coreclr_args = setup_args(main_args)
+
+    diff_summary_dir = coreclr_args.diff_summary_dir
+    arch = coreclr_args.arch
+    platform = coreclr_args.platform
+
+    # Consolidate all superpmi_diff_summary_*.md in overall_diff_summary_<platform>_<architecture>.md
+    # (Don't name it "superpmi_xxx.md" or we might consolidate it into itself.)
+    final_md_path = os.path.join(diff_summary_dir, "overall_diff_summary_{}_{}.md".format(platform, arch))
```

Comment:
We should also print the baseline git-hash here, probably using full url, to make sure that developer knows which baseline was used for comparison.

Reply:
We don't actually have that information here. We could presumably pass it back from the Helix machine, but I think that should be left for future improvement. The baseline can be found in the AzDO log files if actually needed. I presume most people won't need to know the exact baseline hash to repro diffs.